### PR TITLE
Document publicly and publish our roadmap

### PR DESCRIPTION
**Pull request by rouazana**
The main change introduced by this PR is adding this section:
Capture d’écran de 2020-02-27 15-08-46

The first commit is also updating the introduction:
Capture d’écran de 2020-02-27 15-09-00

### CODE REVIEW

File: src/homepage/index.html [hunk 1]
```diff
@@ -126,6 +127,43 @@ <h2>Get Started With James</h2>
...
+                  <li><span class="long-arrow-right">&#8594;</span>Providing a <b>distributed mail server</b>,
+                    fully horizontally scalable.</li>
+                  <li><span class="long-arrow-right">&#8594;</span><span>Maintaining a <b>standalone version</b>
+                    suitable for a single server, based on <b>JPA</b>.</span></li>
+                </ul>
+              </li>
+              <li class="post-template"><span><b>INCOMING FEATURES</b><br>
+                <ul>
+                  <li><span class="long-arrow-right">&#8594;</span><span><b>Blobs garbage collector</b>:
+                    blobs are stored in an object storage mainly for limiting Cassandra hosting cost.
```
Comment (blackheaven) [inline src/homepage/index.html]: I think it's useless since the GC will be able to run on any blobstore.

**Comment by openpaas-james** [comment]
Build finished. No test results found.

File: src/homepage/index.html [hunk 2]
```diff
@@ -72,8 +73,8 @@ <h2>About</h2>
               <p class="align-left">Create your <b>own personal solution</b> of emails treatment by assembling the components you need thanks to the Inversion of Control mail platform offered and  go further customizing filtering and routing rules using <b>James Mailet Container</b>.</p>
               <div class="about-table">
                 <h3 class="tb-h3">James from a technical point of view</h3>
-                <b>Complete portability</b> (100% pure Java)<br/>
-                <b>Built with Java 8</b> and running on the <b>JRE 1.8</b>.<br/><br/>
+                <b>Complete portability</b> (100% pure JVM: Java &amp; Scala)<br/>
+                <b>Built with Java 11</b> and running on the <b>JRE 1.8</b>.<br/><br/>
```
Comment (mbaechler) [inline src/homepage/index.html]: we can still build it with java 8, I would not advertise that.

Comment (mbaechler) [inline src/homepage/index.html]: we can still build it with java 8, I would not advertise that.

Comment (rouazana) [reply src/homepage/index.html]: wrong: if you build it with java8 you get the --release flags error

File: src/homepage/index.html [hunk 3]
```diff
@@ -126,6 +127,43 @@ <h2>Get Started With James</h2>
...
+                    fully horizontally scalable.</li>
+                  <li><span class="long-arrow-right">&#8594;</span><span>Maintaining a <b>standalone version</b>
+                    suitable for a single server, based on <b>JPA</b>.</span></li>
+                </ul>
+              </li>
+              <li class="post-template"><span><b>INCOMING FEATURES</b><br>
+                <ul>
+                  <li><span class="long-arrow-right">&#8594;</span><span><b>Blobs garbage collector</b>:
+                    blobs are stored in an object storage mainly for limiting Cassandra hosting cost.
+                    The deduplication mechanism used to save some spaces makes complicated to remove blobs. We are implementing a
```
Comment (remk) [inline src/homepage/index.html]: s/makes/makes it ?

File: src/homepage/index.html (hunk 2 above)
Comment (mbaechler) [reply src/homepage/index.html]: > wrong: if you build it with java8 you get the --release flags error

I know that but it's a very small detail as you could technically remove the use of this flag an it build with JDK8.

On the other side, we forbid the use of any JDK11 API.

What is the closest description of the reality?

Comment (rouazana) [reply src/homepage/index.html]: currently users are complaining they cannot build James with JDK8. The solution we give them is to use JDK11.

If you have a simple patch to allow James to build on JDK8 I will be happy to integrate it and change my wording.

Comment (mbaechler) [reply src/homepage/index.html]: Just to be sure, to the question

> What is the closest description of the reality?

You think "JDK11" is the best answer.

If so, let's opt for that

**Comment by Arsnael** [comment]
Merged

**Pull request closed without merging**